test/integration/features/profile.feature

```diff
@@ -160,3 +162,28 @@ Feature: Profile
      Then Minishift should have state "Does Not Exist"
      When executing "minishift ip"
      Then exitcode should equal "1"
+
+  Scenario: As user, I cannot create profile when used along with profile <subcommand> --profile name
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift profile list --profile foo" succeeds
+     Then stdout should not contain
+        """
+        foo
+        """
+
+  Scenario Outline: Test all minishift commands along with --profile flag to make sure it doesn't create
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift <commands> --profile foo"
+     Then exitcode should equal "1"
+      And stderr should contain
+      """
+      Profile: foo doesn't exist, Use `minishift profile set foo` or `minishift start --profile foo` to create
+      """
+  Examples: minishift commands
+    | commands  |
+    | ip        |
+    | status    |
+    | delete    |
+    | logs      |
+    | stop      |
+    | version   |
```

Comment:
@budhrg @agajdosi asked a empty line here for one of my PRs. Not sure why.

Reply:
Yes, when we see below mark in Github, we need empty line.
![screenshot from 2017-12-12 15-15-02](https://user-images.githubusercontent.com/1132451/33877768-5d6fc884-df4f-11e7-9bc2-50a09baeb3c5.png)

See reason https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
@LalatenduMohanty I am sorry, my mistake, most of our features and features on other projects do not have a new line at the end, for sure it has no effect on actual functionality